### DISCUSSION TITLE
Scope of DKAN v1 to DKAN v2 migration (in terms of timeline and team-size)

### DISCUSSION THREAD

**Discussion by joepayo** [GetDKAN/dkan#3906] [Migration help]
Hello,
I work for University of Michigan’s Gerald R. Ford School of Public Policy as a Web Developer. I am looking for more information about the upgrade/migration from DKAN 7.x (and Drupal 7) to DKAN v.2. [The Michigan Education Data Center (MEDC) site](https://medc.miedresearch.org/) is the website I'm working with. I was wondering if there were any resources or contacts for another site that may have undergone a similar upgrade recently. For now, **I am looking for what it would take (in terms of scope: team-size and timeline) and what others have typically experienced for a similar site** more than how to accomplish the upgrade.
Thanks for your help,
Joe Payovich
joepayo@umich.edu

**Comment by janette** [comment]
@joepayo the difficulty of the upgrade will depend on how extensive the non-dkan content of your site is and if you need to retain the authorship connections to the content.
